features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -502,6 +544,21 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     }, [ selectedGrantTypes, isGrantChanged ]);
 
+    // useEffect(() => {
+    //     setHybridFlowResponseTypeField(false);
+    //     if (selectedGrantTypes?.includes(ApplicationManagementConstants.AUTHORIZATION_CODE_GRANT)) {
+    //         setHybridFlowResponseTypeField(true);
+    //     }
+    // }, [ selectedGrantTypes, isGrantChanged ]);
```

Comment:
Remove the commented code

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6271/commits/f8f83cbce9604579d623e092f1717a073b8cca11